src/ui/common/src/components/pages/operator/id/index.tsx

```diff
@@ -47,6 +48,19 @@ const OperatorDetailsPage: React.FC<OperatorDetailsPageProps> = ({
     workflowDagResultIdProp
   );
 
+  const { data, error, isLoading } = useGetDagResultQuery(
+    {
+      apiKey: user.apiKey,
+      workflowId,
+      dagResultId: workflowDagResultId,
+    },
+    { pollingInterval: 5000 }
```

Comment:
Neat! So I take it this will continually refetch the DAG every five seconds?

Or is this some sort of retry logic?